proposals/lip-Introduce_cross-chain_update_mechanism.md

```diff
@@ -0,0 +1,588 @@
+```
+LIP:
+Title: Introduce cross-chain update mechanism
+Author: Maxime Gagnebin <maxime.gagnebin@lightcurve.io>
+Discussions-To: https://research.lisk.com/t/introduce-cross-chain-update-mechanism/298
+Type: Standards Track
+Created: <YYYY-MM-DD>
+Updated: <YYYY-MM-DD>
+Requires: Introduce cross-chain messages, 
+          Introduce Interoperability module,
+	  Introduce a certificate generation mechanism,
+	  Introduce BFT module
+```
+
+
+## Abstract
+
+This LIP introduces _cross-chain update transactions_ which are used to post certified information and cross-chain messages in Lisk ecosystem chains. 
+
+Cross-chain update transactions are the carriers of the information transmitted between chains. 
+By posting a cross-chain update, the receiving chain gets the information required about the advancement of the sending chain. 
+The transaction can also include cross-chain messages and thus serves as an envelope for messages from one chain to another.
+
+
+## Copyright
+
+This LIP is licensed under the [Creative Commons Zero 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/).
+
+
+## Motivation
+
+Motivation for the Lisk Interoperability module and the cross-chain update transaction can be found in [LIP "Introduce Interoperability module"][base-interoperability-LIP].
+
+
+## Rationale
+
+
+### Cross-chain Update Transaction Properties
+
+
+#### sendingChainId
+
+Identifies the chain sending the cross-chain update. 
+Only cross-chain update transactions coming from [registered][registration-LIP] chains are valid, registered chains are the ones with an entry in the interoperability store corresponding to their `chainID`.
+
+
+#### certificate
+
+Used to update the cross-chain account. It must be a certificate with an adequate [BLS signature][BLS-LIP]. 
+The signature is always validated against the public key set stored in the sending chain account.
+
+It is important to note here that the validation of this signature is done with respect to the network identifier of the sending chain. 
+This network identifier is included in the sending chain account in the interoperability store.
+
+
+#### activeValidatorsUpdate
+
+The chain account stores an array containing the bls keys and bft weights required to validate the certificate signature. 
+This array has to be updated if the validator set changes in the chain sending the certificate. 
+The difference between the stored array and the new one is included in this property.
+
+
+#### newCertificateThreshold
+
+The chain account stores the threshold required to validate the certificate signature. 
+This certificate threshold has to be updated if its value changes in the chain sending the certificate,
+In that case, the updated value is set in the `newCertificateThreshold` property of the CCU.
+
```

Comment:
Maybe here we could clarify: "If `newCertificateThreshold` is set to 0, it implies that the certificate threshold has not been changed in the sending chain.

Reply:
nice one, added.